app/controllers/v0/sessions_controller.rb

```diff
@@ -95,9 +92,22 @@ def handle_completed_slo
       end
     end
 
-    def log_errors(message)
+    def build_logout_errors(logout_response, logout_request, session, user)
```

Comment:
The one line I added to error building broke the camels back and lint yelled at me for `handle_completed_slo` complexity was too high; so I broke the error building logic into it's own method.